README.md

```diff
@@ -15,324 +15,6 @@ with a few significant changes:
   - Download > Decryption > Blob
 
 ## API Reference
-<a name="module_oyster-streamable"></a>
```

Comment:
@CKH4 do you know why running `yarn docs` deleted this?

Reply:
You added npx to the start of the command. I guess that messed with the arguments?